system.c

```diff
@@ -379,8 +379,11 @@ uint8_t system_execute_line(char *line)
             case 'R':
             {
               const uint8_t result = ad5121_read_pot((enum AD5121_ID)dev_id);
+	      printPgmString(PSTR("[POT<"));
```

Comment:
It's already done in emacs, this is just a byproduct of having multiple configurations and forgetting to switch.